plugins/services/src/js/containers/service-connection/ServiceConnectionEndpointList.js

```diff
@@ -165,6 +166,34 @@ class ServiceConnectionEndpointList extends React.Component {
     });
   }
 
+  getEndpoints(service) {
+    const webUrl =
+      service instanceof Pod ||
```

Comment:
Do pods always have web urls?

Reply:
This page isn't even shown for pods as far as I can tell, for them we use `plugins/services/src/js/containers/service-connection/ServicePodConnectionEndpointList.js`.

Reply:
Good catch.